test/e2e/util.go

```diff
@@ -312,18 +312,19 @@ func logf(level string, format string, args ...interface{}) {
 }
 
 func Logf(format string, args ...interface{}) {
-	logf("INFO", format, args...)
+	msg := fmt.Sprintf(format, args...)
+	logf("INFO %v", msg)
```

Comment:
These changes are wrong. The first parameter of `logf` is a level, not a format string. Unfortunately go vet bases the formatting string analysis off of function names. https://golang.org/cmd/vet/#hdr-Printf_family

I would "fix" this by renaming `logf` to `log`.